rocksdb_replicator/rocksdb_replicator.h

```diff
@@ -63,6 +63,7 @@ struct LogExtractor : public rocksdb::WriteBatch::Handler {
 enum class DBRole {
   MASTER,
   SLAVE,
+  NOOP, // Don't perform any replication with this DB
```

Comment:
Can we just use MASTER for this use case?

Reply:
prefer NOOP because I personally think it is more readable/understandable to mark the db as not meant for replication, but if you prefer MASTER I think it should work. I'm not 100% confident on the code and if it will try to push updates out to slaves, but willing to just go with MASTER if you prefer

Reply:
Master won't push data to slave. It only reactively answer request from slaves. I am ok to go with NOOP, as it can help minimize changes to other use cases.